.pre-commit-config.yaml

```diff
@@ -1,42 +1,9 @@
 repos:
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.13.2
-    hooks:
-      - id: isort
-        stages: [commit]
-  - repo: https://github.com/ambv/black
-    rev: 23.12.1
-    hooks:
-      - id: black
-        args: [--config=backend/pyproject.toml]
-        exclude: "migrations|snapshots"
-        stages: [commit]
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.1.0
-    hooks:
-      - id: flake8
-        args: [--config=backend/.flake8]
-        additional_dependencies: [flake8-bugbear==22.12.6]
-        stages: [ commit ]
-        exclude: /deployment/|/migrations/
-  # mypy precommit hook
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.982
-    hooks:
-      - id: mypy
-        verbose: true
-        args: [--config-file=backend/pyproject.toml, --follow-imports=skip]
-        pass_filenames: true
-        stages: [commit]
-        additional_dependencies: [
-          types-requests==2.28.11.15,
-          types-redis==4.5.1.4,
-          types-python-dateutil==2.8.19.10,
-          types-pytz==2022.7.1.2,
-          djangorestframework-stubs==1.9.1,
-          graphene-stubs==0.15,
-          django-stubs==1.15.0,
-          django-stubs-ext==0.7.0,
-          openpyxl-stubs==0.1.25
-        ]
-        exclude: /deployment/|/migrations/
+  - repo: local
```

Comment:
@johniak  let's add local as an option and keep the rest of the hooks aswell

Reply:
I have some concerns about this approach, as the results from the old hooks were inconsistent with the information available in the CI. Also our whole application stack is based on docker.